docs/src/main/asciidoc/http-reference.adoc

```diff
@@ -556,3 +556,32 @@ link:https://undertow.io/undertow-docs/undertow-docs-2.0.0/index.html#predicates
 === web.xml
 
 If you are using a `web.xml` file as your configuration file, you can place it in the `src/main/resources/META-INF` directory.
+
+== References
+
+=== Well-known route order values
+
+Route order marks are the values that are specified via Vert.x route `io.vertx.ext.web.Route.order(int)` function.
+
+Quarkus uses a couple of route order marks itself. Your route order mark values should be at least a positive integer
```

Comment:
Maybe something like:

```
Quarkus registers a couple of routes with specific order values.
The constants are defined in the `io.quarkus.vertx.http.runtime.RouteConstants` class and listed in the table below.
A custom route should define the order of value 20000 or higher so that it does not interfere with the functionality provided by Quarkus and extensions.
```